what-do-you-do-when-youre-sick.md

```diff
@@ -0,0 +1,37 @@
+# What do you do when you're sick?
+
+We've all been there. You get a cold, take some home remedies, sleep, and it goes away. Or you hurt yourself - ice, heat, rest ... all better, no doctor. You only go to the doctor when (1) you don't know what's going on, (2) the pain is too much to bear, or (3) someone who sees something you don't forces you to go. Either you or someone else must believe you are sick before making the effort to find and go to a doctor.
+
+Let's talk about a different kind of sickness:
+
+> 30 But the Pharisees and the teachers of the law who belonged to their sect complained to his disciples, "Why do you eat and drink with tax collectors and sinners?"
+>
+> 31 Jesus answered them, “It is not the healthy who need a doctor, but the sick. 32 I have not come to call the righteous, but sinners to repentance.”
+>
+> **Luke 5:30-32**
+
+Jesus isn't talking about physical ailments - he's talking about a sickness of the soul. The "hidden" meaning behind what he is saying is that we are *all* sick, but we don't all call it what it is. There are two ways we deem ourselves "healthy":
+
+- We compare our faults to others and figure that ours are not as bad as theirs, so we must be doing ok. The Pharisees, who Jesus was directly addressing in this passage, fell into this camp. Jesus deals with that again in Luke 18:9-14.
+- We make up our own standard of right and wrong and thus are never really in any danger of any kind of moral shortcoming - at least nothing that we couldn't fix ourselves by just changing our minds about something.
+
+Let's talk about that second one for a bit because it is especially true in our world today with moral relativism being the leading belief system. In other words, people believe morality is constructed by society to support our intrinsic common goal of survival and prosperity. That's why things that are wrong to one group of people are ok with another group. With such a belief system, no one is ever really "wrong", certainly not enough to seek out a true cure for the things that lead to our vices.
+
+I wonder if people espousing this belief system have considered the implications of it. For instance, how many of us take anyone seriously when they claim to be a perfect human being? Under this belief system you would be forced to. How about rape - anyone think that's ever not wrong? Well with moral relativism I could argue that because sex is a pleasurable and somewhat chemically driven action, I should not have to abide by whatever society tells me is the "right" way to engage in it … because there is no absolute "right" way remember. You can morally justify rape and many other things under this belief system. It's rather absurd when you really think about all of its implications.
```

Comment:
> Under this belief system you would be forced to
+ (if you're being intellectually honest with yourself)

Perhaps add a footnote explain the logic behind this conclusion.